chainer/functions/evaluation/accuracy.py

```diff
@@ -59,9 +59,15 @@ def accuracy(y, t, ignore_label=None):
     """Computes multiclass classification accuracy of the minibatch.
 
     Args:
-        y (Variable): Variable holding a matrix whose (i, j)-th element
-            indicates the score of the class j at the i-th example.
-        t (Variable): Variable holding an int32 vector of ground truth labels.
+        y (:class:`~chainer.Variable` or :class:`numpy.ndarray` or \
+        :class:`cupy.ndarray`):
+            Array whose (i, j, k, ...)-th element indicates the score of
+            the class j at the (i, k, ...)-th sample.
+            The prediction label :math:`\\hat t` is calculated by the formula
+            :math:`\\hat t_(i, k, ...) = argmax_j y(i, j, k, ...)`.
```

Comment:
I think we usually do not make `argmax` italic. How about changing it to  `\operatorname{\mathrm{argmax}}` ?